README.md

```diff
@@ -1,6 +1,8 @@
 # Libxmljs
 [![Build Status](https://secure.travis-ci.org/polotek/libxmljs.svg?branch=master)](http://travis-ci.org/polotek/libxmljs)
 
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/cf862a98w7qsajpl?svg=true)](https://ci.appveyor.com/project/rchipka/libxmljs)
```

Comment:
Awesome. As long as it works, good with me. Will it automatically build when we push new code?